apps/frontend/src/components/global/Sidebar.vue

```diff
@@ -12,39 +12,37 @@
   >
     <v-list dense class="px-2" subheader>
       <v-subheader>Files</v-subheader>
-      <FileItem v-for="(file, i) in visible_files" :key="i" :file="file" />
-      <v-list-item title="Show all files' controls" @click="toggle_all">
-        <v-list-item-avatar>
-          <v-icon small>mdi-format-list-bulleted</v-icon>
-        </v-list-item-avatar>
-
-        <v-list-item-content>
-          <v-list-item-title>
-            <div v-if="all_toggled">Deselect all reports</div>
-            <div v-else>Select all reports</div>
-          </v-list-item-title>
-        </v-list-item-content>
-      </v-list-item>
-
-      <v-subheader>Views</v-subheader>
-      <v-list-item to="/results" title="View controls' results">
-        <v-list-item-avatar>
-          <v-icon small>mdi-television-guide</v-icon>
-        </v-list-item-avatar>
-
-        <v-list-item-content>
-          <v-list-item-title>Results</v-list-item-title>
-        </v-list-item-content>
-      </v-list-item>
-      <v-list-item to="/compare" title="Compare evaluations' controls">
-        <v-list-item-avatar>
-          <v-icon small>mdi-triangle-outline</v-icon>
-        </v-list-item-avatar>
-
-        <v-list-item-content>
-          <v-list-item-title>Comparison</v-list-item-title>
-        </v-list-item-content>
-      </v-list-item>
+      <v-expansion-panels v-model="file_views" flat>
+        <DropdownContent
+          text="Results"
+          toggle="Select all result set"
+          :files="visible_evaluation_files"
+          route="/results"
+        />
+        <DropdownContent
+          text="Profiles"
+          toggle="Select all profile set"
+          :files="visible_profile_files"
+          route="/profiles"
+        />
+      </v-expansion-panels>
+    </v-list>
+    <v-list dense class="px-2" subheader>
+      <v-subheader>Tools</v-subheader>
+      <slot />
+    </v-list>
+    <v-list dense class="px-2" subheader>
+      <v-subheader>Info</v-subheader>
+      <AboutModal>
+        <template v-slot:clickable="{on}">
+          <LinkItem key="about" text="About" icon="mdi-information" v-on="on" />
+        </template>
+      </AboutModal>
+      <HelpModal>
+        <template v-slot:clickable="{on}">
+          <LinkItem key="help" text="Help" icon="mdi-help-circle" v-on="on" />
+        </template>
+      </HelpModal>
```

Comment:
These were all moved to a new location in the master branch, this PR should not add them back in.